<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="dataSourceStorageLocal" created-in="PY-213.6461.77">
    <data-source name="Django default" uuid="00547b1f-7a83-4cde-aa2e-0d774cc6bb63">
      <database-info product="" version="" jdbc-version="" driver-name="" driver-version="" dbms="MYSQL" exact-version="0" />
      <user-name>root</user-name>
      <schema-mapping />
    </data-source>
    <data-source name="MySQL - devops@localhost" uuid="505c7d71-72ea-4d36-a31c-eaa945ab1c00">
      <database-info product="MySQL" version="8.0.19" jdbc-version="4.2" driver-name="MySQL Connector/J" driver-version="mysql-connector-java-8.0.25 (Revision: 08be9e9b4cba6aa115f9b27b215887af40b159e0)" dbms="MYSQL" exact-version="8.0.19" exact-driver-version="8.0">
        <extra-name-characters>#@</extra-name-characters>
        <identifier-quote-string>`</identifier-quote-string>
      </database-info>
      <case-sensitivity plain-identifiers="mixed" quoted-identifiers="mixed" />
      <secret-storage>master_key</secret-storage>
      <user-name>root</user-name>
      <schema-mapping />
    </data-source>
  </component>
</project>